src/main/java/com/thevoxelbox/voxelsniper/brush/RegenerateChunkBrush.java

```diff
@@ -25,7 +25,7 @@ private void generateChunk(final SnipeData v) {
 
         for (int z = CHUNK_SIZE; z >= 0; z--) {
             for (int x = CHUNK_SIZE; x >= 0; x--) {
-                for (int y = this.getWorld().getMaxHeight(); y >= 0; y--) {
+                for (int y = this.getMaxHeight(); y >= 0; y--) {
```

Comment:
```suggestion
                for (int y = this.getMaxHeight(); y >= this.getMinHeight(); y--) {
```

I think this might make more sense? chunks might extend below Y 0.

Reply:
I didn't dare touch it, but if you think it's fine thats good

Reply:
Didn't look too close, but I _think_ it stores the state of every block, to enable undo's. In that case, we also want to store the ones below 0.

Reply:
This is something for a seperate branch then

Reply:
https://github.com/KevinDaGame/VoxelSniper/pull/7
Will look into this later